app/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,13 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+          package="inaka.com.testapp">
+
+    <application
```

Comment:
According to **Lewis**:
> 
| Id                  | Severity | Category    |
|---------------------|----------|-------------|
| GoogleAppIndexingWarning | Warning | Usability |
 **Message**: App is not indexable by Google Search; consider adding at least one Activity with an ACTION-VIEW intent-filler. See issue explanation for more details..
  **Explanation**: Adds URLs to get your app into the Google index, to get installs and traffic to your app from Google Search..